--- CheckCrystalHDInclude.cmake
+++ CheckCrystalHDInclude.cmake.new
@@ -1,4 +1,4 @@
-if(TARGET_COMMON_DARWIN)
+if(NOT TARGET_WIN32)
   plex_find_header(libcrystalhd/libcrystalhd_if.h ${dependdir}/include)
 
   if(DEFINED HAVE_LIBCRYSTALHD_LIBCRYSTALHD_IF_H)
@@ -14,4 +14,4 @@ if(TARGET_COMMON_DARWIN)
       set(HAVE_LIBCRYSTALHD 1)
     endif(CHECK_CRYSTALHD_VERSION)
   endif(DEFINED HAVE_LIBCRYSTALHD_LIBCRYSTALHD_IF_H)
-endif(TARGET_COMMON_DARWIN)
+endif(NOT TARGET_WIN32)
